platform/log/zephyr/src/mender-log.c

```diff
@@ -33,36 +33,47 @@ mender_log_init(void) {
     return MENDER_OK;
 }
 
+/* IMPORTANT: Please note that the default size of internal Zephyr logger buffer is 1024 bytes; if we need log buffer size 
+              to be more than that there is CONFIG_LOG_BUFFER_SIZE option allowing to extend the buffer size. */
+#define LOG_BUFFER_SIZE_BYTES (128)
```

Comment:
This is a maximum log message length, right? `LOG_BUFFER_SIZE` sounds to me like some memory where logs are stored before being flushed or something like that.

Reply:
Yes, `CONFIG_LOG_BUFFER_SIZE` is a circular buffer for the logs while they wait to be consumed by the backend(s)

Reply:
yup, it's a confusing name